src/sentry/tagstore/snuba/backend.py

```diff
@@ -534,27 +534,16 @@ def get_group_tag_keys_and_top_values(
         return keys_with_counts
 
     def __get_release(self, project_id, group_id, first=True):
-        filters = {"project_id": get_project_list(project_id)}
-        conditions = [["tags[sentry:release]", "IS NOT NULL", None], DEFAULT_TYPE_CONDITION]
-        if group_id is not None:
-            filters["group_id"] = [group_id]
-        aggregations = [["min" if first else "max", SEEN_COLUMN, "seen"]]
-        orderby = "seen" if first else "-seen"
-
-        result = snuba.query(
-            dataset=Dataset.Events,
-            groupby=["tags[sentry:release]"],
-            conditions=conditions,
-            filter_keys=filters,
-            aggregations=aggregations,
-            limit=1,
-            orderby=orderby,
-            referrer="tagstore.__get_release",
-        )
-        if not result:
-            return None
+        orderby = "first_seen" if first else "-last_seen"
+        group_releases = GroupRelease.objects.filter(
+            group_id=group_id,
+            project_id=project_id,
+        ).order_by(orderby)
+        if group_releases:
+            release = Release.objects.get(id=group_releases[0].release_id)
```

Comment:
This will cause an extra query. Let's just do 
```
group_releases = list(GroupRelease.objects.filter(
            group_id=group_id,
            project_id=project_id,
).select_related('release').order_by(orderby)[:1])
```